databricks/conftest.py

```diff
@@ -96,7 +96,7 @@ def add_db(doctest_namespace):
     doctest_namespace["db"] = db_name
 
 
-@pytest.fixture(autouse=os.getenv("KOALAS_USAGE_LOGGER", None) is not None)
+@pytest.fixture(autouse=os.getenv("KOALAS_USAGE_LOGGER", "") != "")
```

Comment:
Is this because "KOALAS_USAGE_LOGGER" can be `None` ??

Reply:
It's just changing the way to check the environment variable to see the same between not-set and empty.

Reply:
Oh I got it, thanks! :)